packages/story-editor/src/components/floatingMenu/elements/karma/more.karma.js

```diff
@@ -0,0 +1,211 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../../karma';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../../../../app/font/defaultFonts';
+import { useInsertElement } from '../../../canvas';
+
+const imageData = {
+  x: 200,
+  y: 0,
+  width: 640 / 2,
+  height: 529 / 2,
+  resource: {
+    id: 10,
+    type: 'image',
+    mimeType: 'image/jpg',
+    src: 'http://localhost:9876/__static__/earth.jpg',
+    alt: 'Earth',
+    width: 640,
+    height: 529,
+    baseColor: '#734727',
+  },
+};
+
+const videoData = {
+  x: 0,
+  y: 0,
+  width: 640 / 2,
+  height: 529 / 2,
+  resource: {
+    width: 640,
+    height: 529,
+    type: 'video',
+    mimeType: 'video/mp4',
+    src: 'http://localhost:9876/__static__/beach.mp4',
+    alt: 'beach',
+  },
+};
+
+const gifData = {
+  x: 0,
+  y: 0,
+  width: 640 / 2,
+  height: 529 / 2,
+  resource: {
+    width: 640,
+    height: 529,
+    type: 'gif',
+    mimeType: 'image/gif',
+    src: 'http://localhost:9876/__static__/<gif-link>.gif',
+  },
+};
+
+const textData = {
+  font: TEXT_ELEMENT_DEFAULT_FONT,
+  content: 'Hello world!',
+  x: 10,
+  y: 20,
+  width: 400,
+};
+
+const shapeData = {
+  x: 48,
+  y: 0,
+  width: 148,
+  height: 137,
+  scale: 100,
+  focalX: 50,
+  focalY: 50,
+  mask: {
+    type: 'heart',
+  },
+  link: null,
+};
+
+const stickerData = {
+  border: null,
+  borderRadius: null,
+  flip: { vertical: false, horizontal: false },
+  focalX: 50,
+  focalY: 50,
+  height: 137,
+  lockAspectRatio: true,
+  opacity: 100,
+  rotationAngle: 0,
+  scale: 100,
+  sticker: { type: 'diyInstagramIcon' },
+  type: 'sticker',
+  width: 137,
+  x: 227,
+  y: 0,
+};
+
+describe('More button', () => {
+  let fixture;
+  let insertElement;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    fixture.setFlags({ floatingMenu: true });
+
+    await fixture.render();
+    await fixture.collapseHelpCenter();
+
+    insertElement = await fixture.renderHook(() => useInsertElement());
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  async function clickMoreButtonForElement({ id }) {
+    const elementFrame = fixture.editor.canvas.framesLayer.frame(id).node;
+
+    // open floating menu
+    await fixture.events.click(elementFrame);
+
+    // click more button
+    await fixture.events.click(fixture.editor.canvas.designMenu.more);
+  }
+
+  it('image element: should focus the size and position panel when the more button is clicked', async () => {
```

Comment:
This is the same test for each element. Not sure if we get much value for testing each one since the button always does the same thing

Reply:
Could we click on the actual items in the library to make this more similar to the real usage instead of using `insertElement` with predefined data?